microsoft-edge/extensions-chromium/getting-started/part2-content-scripts.md

```diff
@@ -228,100 +409,47 @@ The updated `manifest.json` that includes the `content-scripts` and `web_accessi
 }
 ```
 
-The `matches` attribute is set to `<all_urls>`, which means that all files in `content_scripts` are injected into all browser tab pages when each tab is loaded.  The allowed files types that can be injected are JavaScript and CSS.  You also added `lib\jquery.min.js`.  You can copy that file from the [/lib/](https://github.com/microsoft/MicrosoftEdge-Extensions/tree/main/Extension%20samples/extension-getting-started-part2/extension-getting-started-part2/lib) folder of the MicrosoftEdge-Extensions repo.
-
 
 <!-- ------------------------------ -->
-#### Add jQuery
-
-This step has already been done in [jquery.min.js](https://github.com/microsoft/MicrosoftEdge-Extensions/blob/main/Extension%20samples/extension-getting-started-part2/extension-getting-started-part2/lib/jquery.min.js) in the MicrosoftEdge-Extensions repo.
+#### The manifest's `content_scripts` section injects JavaScript into all browser tabs
 
-In the content scripts that you're injecting, plan on using jQuery (`$`).  You added a minified version of jQuery and put it in your extension package as `lib\jquery.min.js`.
+In the `content_scripts` key section, the `matches` attribute is set to `<all_urls>`, which means that `.js` (or `.css`) files that are specified in In the `content_scripts` key section are injected into all browser tab pages when each tab is loaded.  The allowed files types that can be injected are JavaScript (`.js`) and CSS (`.css`).  
```

Comment:
```suggestion
In the `content_scripts` key section, the `matches` attribute is set to `<all_urls>`, which means that `.js` (or `.css`) files that are specified in the `content_scripts` key section are injected into all browser tab pages when each tab is loaded.  The allowed files types that can be injected are JavaScript (`.js`) and CSS (`.css`).  
```

Reply:
result, without "In", and with 'manifest' at start:
```
In the `content_scripts` key section of the manifest file, the `matches` attribute is set to `<all_urls>`, which means that `.js` (or `.css`) files that are specified in the
```